### PR TITLE
refactored deploymentizer lib to take a conf object, not path to file

### PR DESCRIPTION
**Pull request by c1freitas**

### CODE REVIEW

File: test/functional/lib/deploymentizer.spec.js [hunk 1]
```diff
@@ -18,10 +18,12 @@ describe("Deploymentizer", () => {
         process.env.SECRET_PASSWORD = "mypassword";
         process.env.GITHUB_TOKEN = "s@mpler@ndomt0ken";
         fse.mkdirsSync(path.join(os.tmpdir(), "generated"));
+
+				const conf = yield yamlHandler.loadFile("/test/fixture/kit.yaml");
```
Comment (chesleybrown) [inline test/functional/lib/deploymentizer.spec.js]: tabs/spaces? 😕

Comment (chesleybrown) [inline test/functional/lib/deploymentizer.spec.js]: I notice a few other places as well.

Comment (c1freitas) [inline test/functional/lib/deploymentizer.spec.js]: :( I will make a sweep of it.

**Comment by chesleybrown** [comment]
Looking good. A few mixed spaces/tabs though.

**Pull request merged**